src/scss/components/team/_team.scss

```diff
@@ -0,0 +1,86 @@
+.team {
```

Comment:
On my screen, the card appears wider than what I see in the high fidelity mocks. Can you experiment with narrower cards? 

![image](https://user-images.githubusercontent.com/17308542/68424697-623b6c00-0159-11ea-9b4e-da2c56e469f2.png)